.github/workflows/release.yml

```diff
@@ -18,9 +18,9 @@ on:
       - '1.13'
 env:
   release_branch: '1.13'            # e.g. 1.13
-  release_version: '1.13.3'         # e.g. 1.13.1rc1
-  final_release_version: '1.13.3'   # e.g. 1.13.1
-  previous_version: '1.13.2'        # e.g. 1.13.0
+  release_version: '1.13.4rc1'         # e.g. 1.13.1rc1
```

Comment:
We won't do a release candidate so this should be 1.13.4

Reply:
Otherwise the PR looks good to me

Reply:
Oh, except the release notes are missing...

Reply:
Release notes are [here](https://github.com/sympy/sympy/wiki/Release-Notes-for-1.13.4), but are you meaning that they should be included in the PR somehow?

Reply:
When a PR is merged the release notes are transferred from the OP to the release notes page by a bot. This PR in the OP says NO ENTRY but it needs to have the release notes for the backported PRs.

Reply:
I copied and pasted the release notes, but do I need to strip out the issue numbers and the authors?

Reply:
Those are added by the bot when it is merged but I think it is fine to leave them in. Then it would have a reference to both PRs.

Reply:
I did a minor edit, so it's a bit clearer about the original PR #s in the release notes. I also see that `sympy-bot` thinks it will merge these to 1.13.3 release notes, but after looking at the code for the bot, I think that is just a remnant of when the original post was made and that the script doesn't re-run the versioning part. It's going to grab the version number from `release.py`, which is correct, so should update the 1.13.4 page correctly.

I've gone and removed the manual edits I made to the page since I think it is going to add these new entries to the `Changes` section.

Unless you have anything else, I think we're good to merge.

Reply:
Checking back in about this... Thoughts?